docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -18,10 +18,10 @@ This is experimental and may be subject to change.
 - [Differences](#differences)
 
 ## How do I use them?
-Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
-This can act as a drop in replacement for ``plugin.yml``, allowing you to specifically target the Paper platform.
+Similarly to Bukkit plugins, you have to introduce a `paper-plugin.yml` file into your jar resources folder.
+This will not act as a drop in replacement for `plugin.yml`, as some things, as outlined in this guide, need to be declared differently.
```

Comment:
```suggestion
This will not act as a drop-in replacement for `plugin.yml`, as some things, as outlined in this guide, need to be declared differently.
```